data/department.csv

```diff
@@ -1,7 +1,53 @@
 dID,office,title
-001,1,Math
-002,2,Science
-003,3,Business
-004,4,Sociology
-005,5,Kinesiology
-006,6,English
+001,1,AUTOMATED BANK TELLER FACILITY
```

Comment:
Office is a foreign key for room. The value for office should correspond to the room that office is located.